pkg/projectinfo/projectinfo.go

```diff
@@ -32,17 +32,11 @@ var (
 
 func ShortAgentVersion() string {
 	commit := gitCommit
-	if len(gitCommit) > 7 {
-		commit = gitCommit[:7]
-	}
 	return GetAgentName() + "/" + gitVersion + "-" + commit
```

Comment:
how about use `normalizeGitCommit()` to optimize this snippet, then as following:
```
return GetAgentName() + "/" + gitVersion + "-" + normalizeGitCommit(gitCommit)
```

Reply:
The latest code has been commited. Optimize this snippet.